packages/client/src/views/GrantDetailsView.vue

```diff
@@ -611,6 +614,7 @@ export default {
 
   .grant-details-content {
     grid-area: content;
+    /* margin: 24px; */
```

Comment:
_nit: can we remove the commented out margin?_

Reply:
Sure.